src/zope/app/apidoc/ifacemodule/browser.rst

```diff
@@ -475,4 +353,4 @@ Return all utilities that provide this interface.
     'name': u'The Foo',
     'path': '__builtin__.Foo',
     'url': None,
-    'url_name': 'VGhlIEZvbw=='}]
+    'url_name': b'VGhlIEZvbw=='}]
```

Comment:
Ah. It actually is text. That's a leftover from an earlier stage, I guess, and it's being allowed by the normalizer. The prefix can be removed. Good catch!